Source/WebCore/crypto/keys/CryptoKeyEC.h

```diff
@@ -89,7 +92,18 @@ class CryptoKeyEC final : public CryptoKey {
     size_t keySizeInBytes() const { return std::ceil(keySizeInBits() / 8.); }
     NamedCurve namedCurve() const { return m_curve; }
     String namedCurveString() const;
+    
+#if PLATFORM(COCOA) && !PLATFORM(GTK)
+    PlatformECKey platformKey() const { return WTF::switchOn( m_platformKey, [&](const std::unique_ptr<typename std::remove_pointer<CCECCryptorRef>::type, WebCore::CCECCryptorRefDeleter>& options) -> PlatformECKey {
+        return options.get();
+            
+        }, [&](const Vector<uint8_t>& options) -> PlatformECKey {
+        return options;
```

Comment:
This creates an unnecessary memory allocation/copy probably

Reply:
what would be the alternative
/